src/discordbot.py

```diff
@@ -0,0 +1,30 @@
+import discord
```

Comment:
This entire file should be in a separate repo, and you should use poetry to install the pypi package for search-that-hash and import it that way :)

Like we did with the website!